src/foam/comics/v2/userfeedback/UserFeedbackException.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2.userfeedback',
+  name: 'UserFeedbackException',
+  implements: ['foam.core.Exception'],
+  documentation: `
+    In cases where the object is not returned back to client user after a request,
+    a UserFeedbackException will be thrown which will chain the message at the time
+    of the exception to the other feedback messages the object has collected on its
+    path to the dao
```

Comment:
s/dao/DAO/

Reply:
When is the object not returned?

Reply:
In the case of remove, if the call is being intercepted by an approval request it will throw the exception.

But now that im thinking about it since removes do return an FObject, it could instead return the currentObj in the DAO with all the userFeedback from the initial obj that was passed through. Lemme know which one you think works better for remove however, since I remember you said its fine to return the object for put but for removes if I didn't remove it then it should technically be an error.

Reply:
also on an intercepted remove - the exception allows/causes the cache to not remove - which is behaviour you want.

Reply:
Oh yeah this was the original reason ^ so throwing an exception is the intended behaviour for removes